components/PressableLink.tsx

```diff
@@ -0,0 +1,26 @@
+import { Href, Link } from "expo-router";
+import { forwardRef } from "react";
+import { Pressable, PressableProps, Text, View } from "react-native";
+
+type PressableLinkProps = PressableProps & {
+  href: Href;
+  label: string;
+};
+
+const PressableLink = forwardRef<View, PressableLinkProps>(
+  ({ href, label, ...rest }: PressableLinkProps, ref) => {
+    return (
+      <Link asChild href={href}>
+        <Pressable ref={ref} {...rest}>
+          <Text className="text-center font-clash font-semibold uppercase text-white">
```

Comment:
for font related styling I suggest 

`pbk-h7` instead of `font-clash font-semibold` 

although I also suggest going h6 for the heading size